frontend/src/components/core/Block/Block.tsx

```diff
@@ -419,11 +428,24 @@ class Block extends PureComponent<Props> {
     })
   }
 
-  public render = (): ReactNode => (
-    <AutoSizer disableHeight={true}>
-      {({ width }) => this.renderElements(width)}
-    </AutoSizer>
-  )
+  public render = (): ReactNode => {
+    if (this.props.deltaBlock && this.props.deltaBlock.horizontal) {
+      // Create a horizontal block as the parent for columns
+      // TODO: Calculate widths for children? We just pick a big number for now.
+      return (
+        <div className="stBlock-horiz" style={{ display: "flex", gap: "8px" }}>
```

Comment:
Let's style this as a component too! Definitely a cleanup activity that can wait for another PR